docs/builtins/General/base64-decode.md

```diff
@@ -1,31 +1,32 @@
 ## base64-decode
-Use `base64-decode` to decode a `string` from unpadded base64 encoding.
+
+Use `base64-decode` to convert a previously-encoded `string` from unpadded base64 encoding to a string.
 
 ### Basic syntax
 
 To decode a `string` from unpadded base64 encoding, use the following syntax:
 
-`(base64-decode string)`
+```pact
+(base64-decode string)
+```
 
 ### Arguments
 
 Use the following argument to specify the `string` to decode using the `base64-decode` Pact function.
 
 | Argument | Type | Description |
 | --- | --- | --- |
-| `string` | `string` | Specifies the base64 encoded `string` to decode. |
+| `string` | string | Specifies the base64-encoded `string` to decode. |
 
 ### Return values
 
 The `base64-decode` function returns the decoded `string`.
 
 ### Examples
 
-The following example decodes a base64 encoded string in the Pact REPL:
+The following example decodes a base64-encoded "aGVsbG8gd29ybGQh" string to the decoded "hello world!" string in the Pact REPL:
 
 ```pact
-pact>(base64-decode "aGVsbG8gd29ybGQh")
+pact> (base64-decode "aGVsbG8gd29ybGQh")
```

Comment:
perhaps this would be a good time to stress most of the important parts of the alphabet we're using like `/` vs `_`